springfox-schema/src/main/java/springfox/documentation/schema/DefaultModelDependencyProvider.java

```diff
@@ -35,11 +35,8 @@
 import springfox.documentation.spi.schema.EnumTypeDeterminer;
 import springfox.documentation.spi.schema.contexts.ModelContext;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please use explicit imports instead of `*` imports.

Reply:
> Please use explicit imports instead of `*` imports.

Done, fixed imports.

Please do not forget that the target of this pull request is not master branch, I aim to improve 2.9.2. So please create a new branch based on 2.9.2, name it 2.9.3 and then merge this pull request  into 2.9.3, then release 2.9.3.